Nighttime-Operations/Observatory/DIMM/index.rst

```diff
@@ -4,11 +4,51 @@
 DIMM
 ###############
 
-This page contains some common calibrations procedures.
+The differential image motion monitor (DIMM) is a standard instrument to measure astronomical "seeing", which is a metric for the amount of atmospheric turbulence affecting image quality. 
 
-.. toctree::
-    :maxdepth: 2
-    :titlesonly:
-    :glob:
+To operate the DIMM (start up, monitor and shutdown), please follow :ref:`DIMM Startup, Shutdown and Monitor <DIMM-Ops>`
 
-    *
\ No newline at end of file
+The DIMM principle is to produce "twin images" of a bright star by a mask with two apertures at the entrance pupil of the telescope. 
+One of the apertures holds a wedge prism that results in the displacement of an image on the sensor, producing the two mentioned pair spots. 
```

Comment:
... producing a pair of images.

it can be a pair, or two spots, but not a pair of two spots, which would be four spots :)

Reply:
👍